### DISCUSSION TITLE
Closing Window without closing application

### DISCUSSION THREAD

**Discussion by d10sfan** [emilk/egui#1601] [Q&A]
I'm looking into using egui glow and part of the application involves creating a window and then closing it but keeping the rust application alive in the background. 

I'm using the event_loop.run_return and I'm back in control of the application when the window is supposed to be closed but I just see a unresponsive window until the application is completely closed.

I would like to be able to destroy the window instead of waiting for the application to close.

**Comment by emilk** [comment]
Discussion here: https://github.com/emilk/egui/issues/1223
